support/doc/import-youtube.md

```diff
@@ -0,0 +1,57 @@
+# Import viddeos from Youtube guide
+
+If you already own a Youtube channel, you can use a Peertube scripts to import all your videos inside Peertube.
+
+ - [Installation](#installation)
+ - [Usage](#usage)
+
+## Installation
+
+## Prerequisites
+
+You need at least 512MB RAM to run the script.  
+Importation can be launched directly from a peertube server (in this case you already have dependencies installed :+1:) or from a separate server, even a dekstop PC.  
+It is also advised to have maximum bandwith possible as you will download and upload video massively :-)
+
+### Dependencies
+
+If you do not run the script from a Peertube server, you need to follow the steps of the [dependencies guide](dependencies.md).
+
+### Installation
+
+Clone the Peertube repo to get the latest version inside your server:
+
+```
+git clone https://github.com/Chocobozzz/PeerTube.git
+CLONE="$(pwd)/Peertube"
+```
+
+Run ``yarn install``
+```
+cd ${CLONE}
+yarn install
+```
+
+Build server tools:
+```
+cd ${CLONE}
+npm run build:server
+```
+
+
+## Usage
+
+You are now ready to run the script : 
+
+```
+cd ${CLONE}
+node dist/server/tools/import-youtube.js -u "PEERTUBE_URL" -U "PEERTUBE_USER" --password "PEERTUBE_PASSWORD" -y "YOUTUBE_URL"
+```
+
+ - PEERTUBE_URL : the full URL of your peertube server where you want to import, eg: https://videos.lecygnenoir.info
```

Comment:
We could just say this is the url of the PeerTube serveur we want to upload to with our credentials. Since the end user will require an account on that instance, it should be pretty obvious to them.

Also, if you really feel the need to provide an URL example, I think we should point to `https://peertube.cpy.re/` since it's more or less the official one.

Reply:
Good point, I have use my own links to be more clear, people clicking on it being able to see what is asked (I have some difficulties to found my "youtube user" URL as YT changes their rules about it)
I agree than for Peertube URL, it's totally obvious ^^'